spec/unit/basket_item_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative "../../lib/basket_item"
+require_relative "../../lib/product"
+
+RSpec.describe BasketItem do
+  subject(:basket_item) { BasketItem.new(product) }
+  let(:product) { Product.new(name: "Ball", price: 10) }
```

Comment:
you should move price to `let`, and then in test do not write: `60`, just `6*price`